### PR TITLE
Add aerodrome tybg eth

### PR DESCRIPTION
**Pull request by 1932293**
Moo Aero TYBG-WETH

API PR: https://github.com/beefyfinance/beefy-api/pull/1370

Strat: [0xe5c4CD247ac11F4E20583b4D3357627F4411D9A4](https://basescan.org/address/0xe5c4cd247ac11f4e20583b4d3357627f4411d9a4#code)
Vault: [0x89303abD5E328Bd5d09920641a779783A228f412](https://basescan.org/address/0x89303abD5E328Bd5d09920641a779783A228f412#code)

TYBG SAFU:
1. Contracts verified ✓
2. Liquidity ✓
3. No wallet holds more than 5% ✓
4. Emission rate = 0 ✓
5. Token owner is 0x000.. and to me there seems to be no way to rug through the contract ✓

### CODE REVIEW

**Comment by roman-monk** [comment]
slightly concerned about holders. are those EOAs with 2% not the same entity? maybe they got it the same way, not sure what those similar processRoute txs do.
need to add TYBG token in address-book in api

**Comment by 1932293** [comment]
Those look like buys to me. The whole supply was minted and put into LP on sushiswap with 1 ETH so I think the large holders bought sometime at the beginning.

api pr is done (not merged yet) just forgot to post link in here (added to original comment)

File: src/config/vault/base.json [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+    "earnContractAddress": "0x89303abD5E328Bd5d09920641a779783A228f412",
+    "oracle": "lps",
+    "oracleId": "aerodrome-tybg-weth",
+    "status": "active",
+    "createdAt": 1709420692,
+    "platformId": "aerodrome",
+    "assets": ["TYBG", "ETH"],
+    "risks": ["COMPLEXITY_LOW", "IL_HIGH", "MCAP_SMALL", "CONTRACTS_VERIFIED"],
+    "strategyTypeId": "lp",
+    "addLiquidityUrl": "https://aerodrome.finance/deposit?token0=0x0d97f261b1e88845184f678e2d1e7a98d9fd38de&token1=0x940181a94a35a4569e4529a3cdfb74e38fd98631&stable=false",
```
Comment (roman-monk) [inline src/config/vault/base.json:21]: seems wrong token1

Comment (1932293) [reply src/config/vault/base.json:21]: Sorry and thanks for fixing

**Pull request merged**